app/models/task.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :integer          not null, primary key
+#  name        :string
+#  description :string
+#  progress    :integer          default(0)
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  project_id  :uuid
+#
+# Indexes
+#
+#  index_tasks_on_project_id  (project_id)
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  validates_presence_of :name
+  validates_presence_of :description
```

Comment:
I'm not sure it makes sense to have a task without a project. It might be a good idea to ensure that is the case with a validation on presence of the project.

For that matter since your `progress` is an enum where it seems the value should be one of the 3 values, you might want a validation on it's presence as well.